### DISCUSSION TITLE
Is it possible to catch a virus using TERMINAL in Visual Studio Code

### DISCUSSION THREAD

**Discussion by Kivypls** [termux/termux-packages#21809] [Q&A] [answered]
If I download libraries using pip or pkg commands, can I get a virus? I was just hacked on steam recently, I still remember

**Comment by twaik** [comment]
No.

**Reply by Biswa96** [reply]
Probably, not 100% impossible.

(Or, I am too pessimistic after reading many zero-days news)

**Comment by sylirre** [comment, marked as answer]
With `pip` yes, it is possible. Especially if you mistype module name.

Just google for news about malware on PyPI. For example: https://thehackernews.com/2024/09/new-pondrat-malware-hidden-in-python.html

That's a common problem for all community-powered software hosting platforms.
